ui-components/src/page_components/listing/ListingCard.tsx

```diff
@@ -22,10 +25,15 @@ export interface ListingCardProps {
   tableHeaderProps?: ListingCardHeaderProps
   tableProps: ListingCardTableProps
   detailsLinkClass?: string
+  listingID: string
+  addFavorite: unknown
```

Comment:
I think the Travis build is complaining about this. I couldn't remember the proper syntax yesterday, but I think this should be:

`addFavorite: (string) => void`

Similarly for `removeFavorite`.

Reply:
It recommended adding an argument: `(arg0: string) => void`